pkg/pool-manager/pool.go

```diff
@@ -31,7 +33,7 @@ const (
 	RuntimeObjectFinalizerName = "k8s.v1.cni.cncf.io/kubeMacPool"
 	networksAnnotation         = "k8s.v1.cni.cncf.io/networks"
 	networksStatusAnnotation   = "k8s.v1.cni.cncf.io/networks-status"
-	vmWaitConfigMapName        = "kubemacpool-vm-configmap"
+	vmWaitConfigMapName        = names.WAITING_VMS_CONFIGMAP
```

Comment:
Thy do we need this indirection? Can't you just use it from `names`?

Reply:
I will fix it

Reply:
We can use it directly if  I add `.` before the package path, but I think it easier to understand from where constants strings come from if we leave the object that hold them (`names` in this case)

Reply:
You can also just use names.WAITING_VMS_CONFIGMAP in this module to keep it explicit. But if you believe this is more readable, fair enough